### ISSUE TITLE
Upgradable proxy contract 

### ISSUE THREAD

**Issue by stwiname** [dn3010/sylo-ethereum-contracts#61]
Is there a proxy contract to take advantage of the `OwnableUpgradable` usage? I see the testnet deployment doesn't use one at all

**Comment by JCSanPedro** [comment]
Yeah we use Open Zeppelin's upgrade proxy plugin which will automatically deploy the proxy contract https://github.com/dn3010/sylo-ethereum-contracts/blob/master/deploy/00_deploy_phase_two.ts#L83. You can look at the proxy and implementation addresses here https://github.com/dn3010/sylo-ethereum-contracts/blob/master/.openzeppelin/ropsten.json.

Listings Proxy: https://github.com/dn3010/sylo-ethereum-contracts/blob/master/.openzeppelin/ropsten.json
Listings Impl: https://github.com/dn3010/sylo-ethereum-contracts/blob/master/.openzeppelin/ropsten.json#L84

**Comment by JCSanPedro** [comment]
I think the actual proxy address implementation is this https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol
